packages/wrangler/src/__tests__/pages/pages.test.ts

```diff
@@ -24,6 +24,7 @@ describe("pages", () => {
 
 		Commands:
 		  wrangler pages dev [directory] [-- command..]  🧑‍💻 Develop your full-stack Pages application locally
+		  wrangler pages download                        ⚡️ Download settings from your project
```

Comment:
Should this be a hidden command to avoid having to document and support it publicly?

Reply:
that's a good callout.

Reply:
We're planning to document this publicly as part of the launch AFAIK